app/server.go

```diff
@@ -727,11 +735,27 @@ func runSecurityJob(s *Server) {
 	}, time.Hour*4)
 }
 
+func doDiagnosticsIfNeeded(s *Server) {
```

Comment:
I'm confused. This methods is never called. Is there an EE PR?

Reply:
no, just a dumb typo :(
thanks for catching it. fixed